amp/repository/aim/view/viewGlobalSettings.jsp

```diff
@@ -713,54 +713,7 @@ var enterBinder	= new EnterHitBinder('gsSaveAllBtn');
 					                                    		%>
 				                                    		</select>
 				                                    	</c:when>
-				                                    	<c:when test='${type == "t_audit_trial_clenaup"}'>
-				                                    	<%
-				                                    		String peridiodvalues = globalSett
-				                                    												.getGlobalSettingsValue();
-				                                    										int selected = Integer
-				                                    												.parseInt(peridiodvalues);
-				                                    	%>
-															<select styleClass="inp-text" name="gsfValue" >
-															<option value="-1"><digi:trn key="aim:globalSettings:Disabled">Disabled</digi:trn></option>
-															<%
-																for (int k = 30; k <= 90; k += 30) {
-																									if (k == selected) {
-															%>
-				                                    					<option value="<%=k%>" selected="selected"><%=k%>
-				                                    						<digi:trn key="aim:globalSettings:Days"> 
-				                                    							Days
-				                                    						</digi:trn>
-				                                    					</option>
-				                                    						 
-				                                    			<%
-				                                    						 				                                    				} else {
-				                                    						 				                                    			%>
-					                                    				<option value="<%=k%>"><%=k%>
-					                                    					<digi:trn key="aim:globalSettings:Days"> 
-				                                    							Days
-				                                    						</digi:trn>
-					                                    				</option>
-					                                    		<%
-					                                    			}
-					                                    											}
-					                                    		%>
-															</select>
-															<%

```

Comment:
This code displays in global settings the next fire date of the jobs. We should keep this functionality. 
The functionality can be seen when the dropdown of the settings takes a value different from 'Disabled'